app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (NotVaccinatedError,

```

Comment:
Code Style: It's better to use parentheses while importing multiple classes/modules. It makes your code cleaner and easier to read. You can modify your import statement as follows:
python
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```